packages/transport/e2e/tests/bridge.test.ts

```diff
@@ -1,7 +1,7 @@
 // testing build. yarn workspace @trezor/transport build:lib is a required step therefore
 import TrezorLink from '../../lib';
 import messages from '../../messages.json';
-
+import fetch from 'node-fetch';
```

Comment:
```suggestion
import fetch from 'cross-fetch';
```

and devDependency to package.json